For some reason, removing this line prevents this error in reverse dependency vlc:

In file included from /home/builder/.termux-build/vlc/src/modules/codec/gstreamer/gstvlcvideopool.c:27:
In file included from /home/builder/.termux-build/vlc/src/modules/codec/gstreamer/gstvlcvideopool.h:31:
In file included from /data/data/com.termux/files/usr/include/gstreamer-1.0/gst/video/gstvideopool.h:25:
In file included from /data/data/com.termux/files/usr/include/gstreamer-1.0/gst/video/video.h:221:
/data/data/com.termux/files/usr/include/gstreamer-1.0/gst/video/gstvideodmabufpool.h:53:24:
error: use of undeclared identifier 'GstVideoBufferPool'
   53 |     VIDEO_DMABUF_POOL, GstVideoBufferPool)
      |                        ^

Partially reverts https://gitlab.freedesktop.org/gstreamer/gstreamer/-/merge_requests/10411

--- a/gst-libs/gst/video/video.h
+++ b/gst-libs/gst/video/video.h
@@ -218,6 +218,5 @@ G_END_DECLS
 #include <gst/video/video-overlay-composition.h>
 #include <gst/video/videooverlay.h>
 #include <gst/video/video-sei.h>
-#include <gst/video/gstvideodmabufpool.h>
 
 #endif /* __GST_VIDEO_H__ */
